doc/source/index.rst

```diff
@@ -195,6 +195,7 @@ Papers
 
    ray-overview/index.rst
    installation.rst
+   oss-users.rst
 
```

Comment:
Should we put this under the community libraries section at the end? Like "More integrations" or something

Reply:
We can also link to this page from README, intro pages, etc.

Reply:
hmm I think this is a big highlight though (think of like a webpage with endorsements)

Reply:
Maybe we bring the entire libraries section up higher then

Reply:
<img width="1135" alt="Screen Shot 2020-09-13 at 5 58 58 PM" src="https://user-images.githubusercontent.com/4529381/93033369-cf762400-f5ea-11ea-9d5b-b3338d7f1918.png">
currently looks like this. just worried that moving all libraries up will push down Ray Core things too much.